F1WM.IntegrationTests/NewsTests.cs

```diff
@@ -120,11 +120,14 @@ public async Task ShouldGetManyNews()
 			var firstId = 42001;
 			var count = 5;
 
-			var response = await client.GetAsync($"{baseAddress}/news?firstId={firstId}&count={count}");
+			var response = await client.GetAsync($"{baseAddress}/news?firstId={firstId}&countPerPage={count}");
 			response.EnsureSuccessStatusCode();
 
 			var responseContent = await response.Content.ReadAsStringAsync();
-			var newsList = JsonConvert.DeserializeObject<IEnumerable<NewsSummary>>(responseContent);
+			var result = JsonConvert.DeserializeObject<NewsSummaryPaged>(responseContent);
+			Assert.NotNull(result);
```

Comment:
Checks are missing for: RowCount (should be larger or equal to PageSize), CurrentPage (should be 1), and PageCount (should be non-zero)